acceptance_tests/features/steps/common.py

```diff
@@ -1,10 +1,26 @@
-from behave import given
+from behave import given, then
 
+from acceptance_tests.features.pages import sign_in_respondent
 from acceptance_tests.features.pages import sign_in_internal
+from acceptance_tests import browser
+from config import Config
+
+
+@given('the respondent is signed into their account')
+def signed_in_frontstage(context):
+    sign_in_respondent.go_to()
+    browser.driver.find_element_by_id('username').send_keys(Config.RESPONDENT_USERNAME)
+    browser.driver.find_element_by_id('password').send_keys(Config.RESPONDENT_PASSWORD)
+    browser.find_by_id('SIGN_IN_BUTTON').click()
+
+
+@then('the user signs out')
+def signed_out(context):
```

Comment:
Would it be better to add 'frontstage' or 'external' here as we'll likely have an internal sign out.